### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by lengushuru**
**In this PRs the following were implemented**

- Use `onchange()` instead of `onclick()` in task checkbox

### CODE REVIEW

File: src/modules/add-task.js [hunk 1]
```diff
@@ -1 +1,134 @@
...
+                                    <input type="checkbox" class="checkme" checked onclick="CheckMe(${element.index});" id="check${element.index}">
+                    
+                                    <input type="text" value="${element.description}" class="tasks" id="item${element.index}" readonly>                                    <i class="fa-solid uil-ellipsis-v edit " id="edit${element.index}" onclick="editItem(${element.index});"></i>
+                                    <i class="fa-solid fa-trash delete" id="delete${element.index}" onclick="removeItem(${element.index});"></i>
+                                           
+                                        </li>`;
+    } else {
+      liItem += `<li class="task">
+
+                            <input type="checkbox" class="checkme" onclick="CheckMe(${element.index});" id="check${element.index}">
```
Comment (fobadara) [inline src/modules/add-task.js]: [REQUIRED] Kindly make use of the onChange event listener instead of onClick.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @lengushuru, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt


## Required Changes ♻️
_Check the comments under the review._
I had to include some comments here because they couldn't be included inline:
[REQUIRED] Your README file is repeated twice. After the License, it starts again from the logo
[REQUIRED] Your live demo link is broken. Kindly fix it

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/add-task.js [hunk 2]
```diff
@@ -1 +1,134 @@
...
+                                    <input type="checkbox" class="checkme" checked onchange ="CheckMe(${element.index});" id="check${element.index}">
+                    
+                                    <input type="text" value="${element.description}" class="tasks" id="item${element.index}" readonly>                                    <i class="fa-solid uil-ellipsis-v edit " id="edit${element.index}" onclick="editItem(${element.index});"></i>
+                                    <i class="fa-solid fa-trash delete" id="delete${element.index}" onclick="removeItem(${element.index});"></i>
+                                           
+                                        </li>`;
+    } else {
+      liItem += `<li class="task">
+
+                            <input type="checkbox" class="checkme" onclick="CheckMe(${element.index});" id="check${element.index}">
```
Comment (MaganaAsiati) [inline src/modules/add-task.js]: - [x] Kindly use onChange instead of onClick as required by the previous reviewer to make you project complete :+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @lengushuru 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mwanawabangona — APPROVED** [review]
Hi @lengushuru 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  

H
## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**